Specs/DateTime/Dutch/SetExtractor.json

```diff
@@ -92,7 +108,9 @@
     "Results": [
       {
         "Text": "15:00 elke dag",
-        "Type": "set"
+        "Type": "set",
+        "Start": -1,
```

Comment:
I will merge this PR.
But seems we don't support these kinds of cases for now, we only support extract text which is part of the original query.

Reply:
Resolved in #1115